BrightCoders-Institute/proyecto-final-jdg
# Add icon user profile navbar

# ISSUE #40  Add Icon and User Profile View in Navigation Bar

# Description

This pull request adds a user profile feature to the application. Users can now view their address information and a list of their previous orders on their profile page.

# Changes Made

- Created a user profile view that displays the user's address information.
- Added a list of previous orders in the user profile view.
- Modified the necessary routes and controllers to support the user profile view.
- Fixed style errors in the mobile view of the address list.
- Modified the style of the address table to display correctly.
- Corrected the error of the delete addresses button. Now it functions correctly.
- Added order ID, order date, and order status fields to the order details view.

# How to test

- Log in as a user.
- Go to your profile by clicking on the user icon in the top right corner.
- Verify that address information is displayed correctly.
- Verify that the list of previous orders is displayed with basic details.
- Ensure that the links to previous orders are functional.

## Screenshots
###Desktop view
![image](https://github.com/BrightCoders-Institute/proyecto-final-jdg/assets/6729543/c0fdaf14-3d2a-4472-a94d-3d7eccb00fbe)



### Mobile view
![image](https://github.com/BrightCoders-Institute/proyecto-final-jdg/assets/6729543/21a3a341-0f9c-4950-b316-42d7e57b077a)
